litestar/app.py

```diff
@@ -205,6 +205,7 @@ def __init__(
         return_dto: type[AbstractDTO] | None | EmptyType = Empty,
         security: Sequence[SecurityRequirement] | None = None,
         signature_namespace: Mapping[str, Any] | None = None,
+        signature_types: Sequence[Any] | None = None,
```

Comment:
Couldn't the type be `Sequence[type[Any]]`?

Reply:
Not all valid type-annotations are types though, and this is consistent with how we have typed `signature_namespace`.

```py
>>> from inspect import isclass
>>> from typing import Union
>>> isclass(Union)
False
```